pkgs/stdenv/generic/setup.sh

```diff
@@ -1283,6 +1283,7 @@ genericBuild() {
     fi
 
     for curPhase in $phases; do
+        if [[ "$curPhase" = configurePhase && -n "${dontConfigure:-}" ]]; then continue; fi
         if [[ "$curPhase" = buildPhase && -n "${dontBuild:-}" ]]; then continue; fi
```

Comment:
Well we have `doCheck` so there's this interesting use of polarities here.
I'm under the impression that `doCheck` exists to enable because it's default skipped.
Other phases should not be skipped and be explicitly disabled otherwise.

So I'd think `dontUnpack` would be the only other one to complete this.